code/modules/hydroponics/seeds.dm

```diff
@@ -650,3 +650,21 @@
 	reagents_from_genes()
 
 	return TRUE
+
+/*
+ * Both `/item/food/grown` and `/item/grown` implement a seed variable which tracks
+ * plant statistics, genes, traits, etc. This proc gets the seed for either grown food or
+ * grown inedibles and returns it, or returns null if it's not a plant.
+ *
+ * Returns an `/obj/item/seeds` ref for grown foods or grown inedibles.
+ *  - returned seed CAN be null in weird cases but in all applications it SHOULD NOT be.
+ * Returns null if it is not a plant.
+ */
+/obj/item/proc/get_plant_seed()
```

Comment:
ehhh its not great but its better at least. should probably just combine the type paths at some point but, out of scope for this pr

Reply:
I've been theorizing ways to combine the two plant types without doing something like making a subtype of food that's inedible for a while, I agree with you there completely.

Reply:
You don't need things to be subtypes of food, just apply the edible component.

`/obj/item/food` is good because it's essentially just a wrapper for the edible component. It provides no behavior of its own, and so should be identical.